sdks/java/io/jms/src/main/java/org/apache/beam/sdk/io/jms/JmsRecord.java

```diff
@@ -31,8 +30,8 @@
   private final String jmsMessageID;
   private final long jmsTimestamp;
   private final String jmsCorrelationID;
-  private final Destination jmsReplyTo;
-  private final Destination jmsDestination;
+  private final String jmsReplyTo;
+  private final String jmsDestination;
```

Comment:
`Destination` can't be nullable, but `ReplyTo` can. I will update accordingly. Thanks !